src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -49,38 +55,77 @@ public UserController(UserRepository userRepository, BookRepository bookReposito
         this.bookRepository = bookRepository;
     }
 
+    @ApiOperation(value = "find all users")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Succesfully retrieved users", responseContainer = "List",
+            response = User.class)
+    })
     @GetMapping
     public List<User> findAll() {
         return userRepository.findAll();
     }
 
+    @ApiOperation(value = "find user by its id", response = User.class)
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Succesfully retrieved user"),
+        @ApiResponse(code = 400, message = "Malformed body request"),
+        @ApiResponse(code = 404, message = "The resource is not found")
```

Comment:
you can extract all messages from responses to constants

Reply:
You're right. I'll refactor it.